CONTRIBUTING.md

```diff
@@ -26,64 +26,15 @@ This is my commit message
 Signed-off-by: Random J Developer <random@developer.example.org>
 ```
 
-Git even has a -s command line option to append this automatically to your
-commit message:
-
-```
-git commit -s -m 'This is my commit message'
-```
-
-If you have already made a commit and forgot to include the sign-off, you can
-amend your last commit to add the sign-off with the following command, which
-can then be force pushed.
-
-```
-git commit --amend -s
-```
-
-## Getting Started
-
-1. Fork the repository on GitHub
-1. Read the [install](docs/install.md) guide for build and deploy instructions
-1. Play with the project, submit bugs, submit patches!
-
-## Contribution Flow
-
-This is a rough outline of what a contributor's workflow looks like:
-
-1. Create a branch from where you want to base your work (usually main).
-1. Make your changes and arrange them in readable commits.
-1. Make sure your commit messages are in the proper format (see below).
-1. Push your changes to the branch in your fork of the repository.
-1. Make sure all [tests](docs/testing.md) pass, and add any new [tests](docs/testing.md)
-   as appropriate.
-1. Submit a pull request to the original repository.
-
 ## Coding Style
 
-Ramen project is written in golang and follows the style guidelines dictated
-by the go fmt as well as go vet tools.
+Ramen project is written in golang and follows the style guidelines dictated by
+the go fmt as well as go vet tools.
```

Comment:
This was good if it was true, actually we let random contriuotors to golangci-lint dictate the style for us :-)

Reply:
I agree but I am leaving this content same as the previous one. Just moved it a bit. We can rework this in a different PR.

Reply:
Please see sentence below "Additionally various golang linters are enforced to ensure consistent coding"